scripts/test_cover.sh

```diff
@@ -1,9 +1,17 @@
 #!/usr/bin/env bash
 set -e
 
-PKGS=$(go list ./... | grep -v '/simapp')
```

Comment:
Why did you remove simapp? Is it just legacy?

Reply:
yeah, we don't have a simapp directory here